en/Obsidian/Obsidian.md

```diff
@@ -55,6 +55,6 @@ We support [[Customizing CSS|custom CSS]] and have a [[Community plugins|beta pl
 
 ## Follow our development
 
-Obsidian is in public beta right now. We have [a roadmap](https://trello.com/b/Psqfqp7I/obsidian-roadmap) that you can check out.
+Obsidian is stable and public available. We still have grand plans to continue making Obsidian the best and most refined thought-processing app for decades to come. On the [roadmap](https://trello.com/b/Psqfqp7I/obsidian-roadmap) you can see where we are headed.
```

Comment:
```suggestion
Curious how we continue to make Obsidian the best and most refined tool for thought-processing? Check out our [roadmap](https://trello.com/b/Psqfqp7I/obsidian-roadmap) to see what we're working on.
```
 
- Now that we're out of beta, I think we can skip the first part. Think this section should focus on looking ahead. 
- Last part is mostly a suggestion to avoid "where we're", for better flow.

Reply:
I don't think "public available" makes sense. I think it reads better as "publicly available"

Reply:
@claremacrae I agree, though I assume your comment is on the original text, as my suggestion removes the text altogether.

Reply:
Odd. It showed up in the email I was replying too... Never mind!